Modules/Test/classes/tables/class.ilTestQuestionsTableGUI.php

```diff
@@ -15,6 +15,7 @@
 
 class ilTestQuestionsTableGUI extends ilTable2GUI
 {
+    private const CLASS_PATH_FOR_EDIT_LINKS = ['ilrepositorygui', 'ilobjquestionpoolgui'];
```

Comment:
I prefer to use the `::class` constants, but this is just personal taste. Maybe this could be also problematic because a `array_map('strtolower', self::CLASS_PATH_FOR_EDIT_LINKS)` has to be applied then.

Reply:
I am not 100% sure if this route is correct. Whether this route could be resolved depends on the value of `$target_class`. IMO this should be thoroughly tested. If you already tested the 3 occurances of this table, I am fine with this :-).

Reply:
Jup and jup:
1. I generally would agree with you that ```::class``` constant is more elegant. I will change that, as
2. I completely missed the corrections class! Thank you very much for the close look. Update follows...